ogr/services/pagure/pull_request.py

```diff
@@ -267,6 +267,9 @@ def get_statuses(self) -> List[CommitFlag]:
         self.__update()
         return self.project.get_commit_statuses(self._raw_pr["commit_stop"])
 
+    def get_flags(self) -> List[Dict]:
+        return self.__call_api("flag", method="GET")["flags"]
```

Comment:
Using `GET` is the default, but I am not against explicitness since it's deep in the call chain (in `PagureService.call_api_raw`).